brain.py

```diff
@@ -148,6 +145,8 @@ def load_brain():
             activity.face.say_notification(hello)
         elif sorry:
             activity.face.say_notification(sorry)
+        else:
+            activity.face.say_notification(_("Hi again! I'm %s.") % brain_name)
 
     GLib.idle_add(load_brain)
-    return True
+    return True
```

Comment:
Please remove the no new line at EOF, you can do so with `echo >> brain.py`, take a look at git diffs before making commits, it'll help with preventing issues like this.